_docs/installation.md

```diff
@@ -14,7 +14,7 @@ gem install kamal
 ...otherwise, you can run a dockerized version via an alias (add this to your .bashrc or similar to simplify re-use). On macOS, use:
 
 ```sh
-alias kamal="docker run -it --rm -v '${PWD}:/workdir' -v '/run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock' -e SSH_AUTH_SOCK='/run/host-services/ssh-auth.sock' -v /var/run/docker.sock:/var/run/docker.sock ghcr.io/basecamp/kamal:latest"
+alias kamal='docker run -it --rm -v "${PWD}:/workdir" -v "/run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock" -e SSH_AUTH_SOCK="/run/host-services/ssh-auth.sock" -v /var/run/docker.sock:/var/run/docker.sock ghcr.io/basecamp/kamal:latest'
```

Comment:
@zupolgec thank you for the fix! 

I've identified another issue with this alias that is not directly related to the problem you are addressing. However, resolving it may involve introducing another expansion here. The location of the socket can vary across systems. In case of Linux, it is discoverable through the `$SSH_AUTH_SOCK` variable and is located at `/run/user/1000/keyring/ssh`, not `/run/host-services/ssh-auth.sock` like in macOS.

The `$SSH_AUTH_SOCK` variable should also be available in macOS. Having it used here might unify the alias for both macOS and Linux.

```suggestion
alias kamal='docker run -it --rm -v "${PWD}:/workdir" -v "${SSH_AUTH_SOCK}:/run/host-services/ssh-auth.sock" -e SSH_AUTH_SOCK="/run/host-services/ssh-auth.sock" -v /var/run/docker.sock:/var/run/docker.sock ghcr.io/basecamp/kamal:latest'
```

Not sure if I should open a separate PR to propose the change. WDYT?